control/pure_pursuit/src/pure_pursuit/pure_pursuit_lateral_controller.cpp

```diff
@@ -109,32 +122,171 @@ void PurePursuitLateralController::setInputData(InputData const & input_data)
   current_steering_ = input_data.current_steering_ptr;
 }
 
+TrajectoryPoint PurePursuitLateralController::calcNextPose(
+  const double ds, TrajectoryPoint & point, AckermannLateralCommand cmd) const
+{
+  geometry_msgs::msg::Transform transform;
+  transform.translation = tier4_autoware_utils::createTranslation(ds, 0.0, 0.0);
+  transform.rotation =
+    planning_utils::getQuaternionFromYaw(((tan(cmd.steering_tire_angle) * ds) / param_.wheel_base));
+  TrajectoryPoint output_p;
+
+  tf2::Transform tf_pose;
+  tf2::Transform tf_offset;
+  tf2::fromMsg(transform, tf_offset);
+  tf2::fromMsg(point.pose, tf_pose);
+  tf2::toMsg(tf_pose * tf_offset, output_p.pose);
+  return output_p;
+}
+
+void PurePursuitLateralController::setResampledTrajectory()
+{
+  // Interpolate with constant interval distance for lateral acceleration calculation.
```

Comment:
Ah sorry for this comment, it should be removed. Because module does not know the sampling distance of output of the planning and does not interpolate the vehicle position and the target point, I think we should ensure that distance between the trajectory points are small enough. 

Also I can interpolate the vehicle position and target point into trajectory. What do you think?